spec/controllers/collection_items_controller_spec.rb

```diff
@@ -321,6 +321,43 @@
         end
       end
     end
+
+    context "as an archivist" do
+      let(:archivist) { create(:archivist) }
+      let(:work) { create(:work) }
+
+      let(:collection) do
+        participant = create(:collection_participant, pseud: archivist.default_pseud)
+        create(:collection, collection_participants: [participant])
+      end
+
+      let(:params) do
+        {
+          collection_names: collection.name,
+          work_id: work.id
+        }
+      end
+
+      before do
+        fake_login_known_user(archivist)
+      end
+
+      context "when the item's creator does not allow collection invitations" do
+        it "adds the item anyway" do
+          post :create, params: params
+          it_redirects_to_simple(work)
```

Comment:
Does the archivist not get any sort of confirmation message here that we could check? If not, we might need an issue for that.

Reply:
They do get a flash message, and I've added a check for that. I went through a few iterations of the implementation, so maybe that got lost